### ISSUE TITLE
Problem beim Ausführen

### ISSUE THREAD

**Issue by Titanflex91** [NilsOmDev/Fireboard_API_Task_Generator#1]
Hallo,

ist das Script noch aktuell ? Denn bei ausführen mit python3 Main.py kommen ein paar Fehler ??

**Comment by Titanflex91** [comment]
Wenn ich die Main.py in CMD aufrufe kommt 

![image](https://github.com/NilsOmDev/Fireboard_API_Task_Generator/assets/86373759/ff243674-f627-4116-b96a-0caeb2044bfe)

**Comment by NilsOmDev** [comment]
Scheint als wenn du Python2 anstatt von Python3 verwendest.
Überprüfe die Version mit 'python --version'.

Ansonsten wirst du fehlende Module sowie 'tabulate' per pip installieren müssen.

**Comment by Titanflex91** [comment]
Die Version ist 3.12.3 

Okay dann muss ich mal schauen wie man das installiert

**Comment by NilsOmDev** [comment]
Version 3.12.3 funktioniert.
Sofern pip installiert ist: 'pip install tabulate'.

**Comment by Titanflex91** [comment]
Okay, also ich habe es nun geschafft das es sich öffnet ... allerdings kommt beim exit folgendes 

![image](https://github.com/NilsOmDev/Fireboard_API_Task_Generator/assets/86373759/136a6243-ff92-458c-a9f2-ce0d6c284090)

**Comment by NilsOmDev** [comment]
Du hast den Exit per Strg+C ausgeführt, was nicht weiter bedenklich ist. 
Um das Programm ohne Error zu schließen, tippe in der Befehlseingabe 'exit' ein und drücke Enter.

**Comment by Titanflex91** [comment]
Hallo, habe es nun mal mit dem Auth Key probiert. In Fireboard Portal kommt nichts an ... 

Grüße

**Comment by NilsOmDev** [comment]
Kommt denn keine Fehlermeldung mehr?

Überprüfe bitte ob das Senden an Fireboard auch tatsächlich aktiviert ist. Es gibt im Skript eine Zeile die aktiviert sein muss.

**Comment by zauberflorian112-ux** [comment]
Aufprobiert läuft, keine direkte .exe Datei gefunden. Es überträgt es noch nicht in Fireboard. Log Datei ist fast leer.

**Comment by zauberflorian112-ux** [comment]
Uns wird es definitiv weiterhelfen, gibt es eine Möglichkeit das sich die Einsätze nach ner bestimmten selbst erstellen ( Realität)

PS: Ich finde es schön das sie sowas kostenlos frei zugänglich machen!

**Comment by Titanflex91** [comment]
Ja das kann ich nur bestätigen. 

Das mit der automatische Erstellen von Einsätzen auf eine Zeit x wäre schön, vielleicht kommt das bei Gelegenheit. 

Da könnte man sich auch überlegen, ob man die Zeit separat einstellen kann.

**Comment by zauberflorian112-ux** [comment]
Jo, wäre vorteilhaft um es realitätsnah dazustellen. Wenn es nicht zu kompliziert ist.

**Comment by NilsOmDev** [comment]
Ein Timer ist jetzt eingebaut unter Release V1.1
